M2/Macaulay2/packages/Complexes/ChainComplexDoc.m2

```diff
@@ -2341,10 +2351,8 @@ doc ///
    Usage
      D = C1 ** C2
    Inputs
-     C1:Complex
-       or @ofClass Module@
-     C2:Complex
-       or @ofClass Module@
+     C1:{Module}
+     C2:{Module}
```

Comment:
I think you meant to also add `Complex` to these lists, no? Otherwise you could just write `C1:Module`. Currently the documentation would not mention that the inputs can be complexes (which I would have thought would signal an error)
```suggestion
     C1:{Module,Complex}
     C2:{Module,Complex}
```

Reply:
Since the first key is e.g. (Hom, Complex, Complex), writing C1:Module gives an error, and it turns out that using {Complex,Module} as suggested, adds in an extra Complex in the doc line.  I'm not sure if that was the planned behavior (not sure when this got added, actually).  We wanted the doc to add extra types on top of Complex, and this seemed to be the way that worked. Perhaps that should be revisited, but probably not for this PR.

Reply:
Wait, sorry, it seems like with what you have the documentation _does_ mention module and complex, but this is a bug that I will fix before the next release! You should still definitely list all possible input types here, so for instance `C1:{Module,Complex}` (even though right now it will probably say something silly like "a complex, a module, or a complex" or something like that).

Reply:
Yes, that's exactly what it says! (Up to order).  How will you fix this bug?  Currently, it does some checking of types (like C:Module would fail in the above case).  How are you planning on modifying these checks?  (one way: make sure the types in the first key actually appear in the list, but allow others?)

Reply:
I opened an issue here: https://github.com/Macaulay2/M2/issues/3465.

For now, please add `Complex`, otherwise when `SimpleDoc` is fixed the documentation will not mention complexes anymore.

Reply:
I need to look at the code to figure out what's the best fix, but the current behavior is definitely wrong and will signal an error.

Reply:
I'll make the changes.